### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by WolferBoi14** [CodenameCrew/CodenameEngine#275] [Q&A]
I keep getting errors that say "process creation failure: attrib +h .temp" what do i do?

**Comment by NexIsDumb** [comment]
okay about that, we fixed it, we'll release the fix in the next update

**Reply by WolferBoi14** [reply]
... And when is the next update?

**Reply by NexIsDumb** [reply]
> ... And when is the next update?

as soon as i finish doing my cool big thing, trust me is gonna be great (almost done)

**Reply by NexIsDumb** [reply]
until then you can use an older build

**Reply by WolferBoi14** [reply]
What older build will work?

**Reply by NexIsDumb** [reply]
> What older build will work? 

hmm, some days earlier than the april fools update I dont remember tbh
try a week or two older

**Reply by WolferBoi14** [reply]
Not working still
